test/sagemaker_tests/pytorch/training/integration/sagemaker/test_pytorchddp.py

```diff
@@ -66,3 +66,35 @@ def test_pytorchddp_throughput_gpu(
         invoke_pytorch_estimator(
             ecr_image, sagemaker_regions, estimator_parameter, job_name=job_name
         )
+
+@pytest.mark.processor("gpu")
+@pytest.mark.model("N/A")
+@pytest.mark.multinode(2)
+@pytest.mark.integration("pytorchddp")
+@pytest.mark.parametrize(
+    "efa_instance_type", get_efa_test_instance_type(default=["ml.p4d.24xlarge"]), indirect=True
+)
```

Comment:
Why do we need to run this test using a p4d.24xlarge, and not, say a p3.8xlarge/16xlarge?

Reply:
Changed!